fastai/tabular/core.py

```diff
@@ -130,7 +130,7 @@ def df_shrink(df, skip=[], obj2cat=True, int2uint=False):
 # Cell
 class _TabIloc:
     "Get/set rows by iloc and cols by name"
-    def __init__(self,to): self.to = to
+    def __init__(self,to): store_attr()
```

Comment:
not sure the point of this change?

Reply:
Yeah thinking back on this, `store_attr()` is really only useful when it's 2+ params. Will revert back